java/src/main/java/com/google/oak/util/Result.java

```diff
@@ -27,15 +28,18 @@
  * fail and produce an error of type {@code E}. An instance of {@code Result}
  * can encapsulate both outcomes.
  *
- * <p>The main invariant that instances of this class maintain is that each object either
+ * <p>
```

Comment:
Now sure why autoformatter always adds this letter. What does it do?

Reply:
It is an html tag for formatting of the generated java doc.